packages/hdwallet-native/src/binance.ts

```diff
@@ -88,8 +91,7 @@ export function MixinNativeBinanceWallet<TBase extends core.Constructor<NativeHD
         if (!tx.sequence) tx.sequence = "0";
         if (!tx.source) tx.source = "0";
 
-        const bnbSdk = await import("bnb-javascript-sdk-nobroadcast");
```

Comment:
isn't this import already lazy because it doesn't get called until someone calls `binanceSignTx`?

Reply:
Yeah, just following the same pattern as the others.